modules/nw-sctp-about.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-about_{context}"]
+= Support for Stream Control Transmission Protocol (SCTP) on {product-title}
```

Comment:
The docs don't expand "TCP" and "UDP", so I wouldn't expand "SCTP" either. People *always* just refer to it by the acronym. (I am sure that most of our customers who have demanded SCTP support could not actually tell you what "SCTP" stands for. I'm not sure I would have guessed correctly if I hadn't just read it... I might have gone with "Sequence" rather than "Stream"...)

Reply:
@danwinship, that makes sense; I may have been overzealous in expanding acronyms. But for SCTP, I think it may be valuable for search engine traffic.